apphost/src/runtime.rs

```diff
@@ -163,4 +166,36 @@ pub fn launch_app_binary(runtime_dir: Option<&Path>, app_info: &AppInfo) -> Resu
         Ok(err) => Err(Box::new(err)),
         Err(_) => Ok(res)
     }
-}
\ No newline at end of file
+}
+
+
+#[cfg(unix)]
+use libc::{sigaction, SIGSEGV, SIG_DFL, sigaltstack, stack_t, SS_DISABLE};
+
+#[cfg(unix)]
+use std::{mem, ptr};
+
+#[cfg(unix)]
+fn prelaunch_setup() {
+    //On Unix it is required for us to manually remove the registered signal handler altstack
```

Comment:
[WARNING: incredibly nitpicky comment ahead]
Maybe split the comment into three different sentences, once at `stable runtime, this` (split the part after this into its own sentence), and once at `structure, see` (moving the `see <link>` part into brackets at the end).